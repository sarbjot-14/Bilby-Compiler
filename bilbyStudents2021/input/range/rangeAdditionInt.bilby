main{

	
    mut myRange := <4 .. 7> + 2;
    
    print myRange $n;
    
    mut myRangeRight :=   2 + <14 .. 101>;
    
    print myRangeRight $n;
  
   
  

}