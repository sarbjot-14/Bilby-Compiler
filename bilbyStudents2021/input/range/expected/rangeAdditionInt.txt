<6..9>
<16..103>
